tests/network/expose.go

```diff
@@ -141,6 +141,45 @@ var _ = SIGDescribe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:c
 		return tests.WaitForJobToSucceed(job, time.Duration(120)*time.Second)
 	}
 
+	executeVirtctlExposeCommand := func(ExposeArgs []string) error {
+		virtctl := tests.NewRepeatableVirtctlCommand(ExposeArgs...)
+		return virtctl()
+	}
+
+	getService := func(namespace, serviceName string) (*k8sv1.Service, error) {
+		svc, err := virtClient.CoreV1().Services(namespace).Get(context.Background(), serviceName, k8smetav1.GetOptions{})
+		if err != nil {
+			return nil, err
+		}
+
+		return svc, nil
+	}
+
+	getAndValidateClusterIp := func(namespace, serviceName string, ipFamily ipFamily) (*k8sv1.Service, error) {
```

Comment:
I think this function does too many things, wouldn't it be enough to return only `error`?
Maybe the service could be passed as parameter instead of `serviceName`.

Reply:
[Done](https://github.com/kubevirt/kubevirt/compare/f1c748b0ac9e6cd2af854a3de1bc92f31ef987eb..1c04a6341c41c4b079ef9f87928c9e2d20f317ad).
So now there are two functions one for getting the server, and another for validating the cluster IP.